src/examples/scala/containers.scala

```diff
@@ -0,0 +1,13 @@
+object containers extends loamstream.LoamFile {
+  val fileIn = store("fileIn.txt").asInput
+  val fileOut1 = store("fileOut1.txt")
+  val fileOut2 = store("fileOut2.txt")
+  
+  drm {
+    cmd"cp $fileIn $fileOut1"
+  }
+  
+  drmWith(imageName = "docker://library/ubuntu:18.04") {
+    cmd"cp $fileIn $fileOut2"
+  }
```

Comment:
Given that these are meant to be examples, comments would be very much appreciated for us Loam noobs. While I can conceptually see the difference between `drm` and `drmWith`, I don't know if these are mutually exclusive examples, or if this is an example doing the same thing twice... in serial? parallel? independent of each other? etc.

Reply:
These are fair questions, but I'm going to resolve most the examples-related comments, as improving the examples beyond porting them to plain Scala is really out of scope for this PR.

A lot of the things you bring up are really asking for Loam-101 documentation.  That's lacking, and would be really good to have, but documentation has never been a priority, unfortunately.  If we ever do a documentation push, improving the examples would be a top priority.